sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -604,4 +606,25 @@ class AnalysisSuite extends AnalysisTest with Matchers {
       checkAnalysis(input, expected)
     }
   }
+
+  test("SPARK-25691: AliasViewChild with different nullabilities") {
```

Comment:
hmm, this test actually comes from another PR`SPARK-25691`. So the original PR doesn't backport too?

Reply:
It got pulled in by the cherry pick because your original PR patched it, and I didn't see harm in the added test. However, its title references a patch that was not pulled in, which is weird, so I should probably remove it.

Reply:
Yep. Please remove this `test("SPARK-25691...")` since this is SPARK-28156 backport.